lms/djangoapps/gradebook/management/commands/tests/test_generate_gradebook_entries.py

```diff
@@ -133,6 +139,7 @@ def get_module_for_user(self, user, course, problem):
             course.id
         )._xmodule
 
+    @skip
```

Comment:
@mattdrayer Seems to have been a side effect from a commit I re-applied that I initially had left out.